mne_bids_pipeline/_report.py

```diff
@@ -848,9 +848,9 @@ def _add_raw(
     cfg: SimpleNamespace,
     report: mne.report.Report,
     bids_path_in: BIDSPath,
+    raw: BaseRaw,
```

Comment:
This change should avoid the problem we saw here, as it now forces us to provide pre-loaded raw data (with hopefully the correct metadata set). previously this was optional. see also the changes below